OpenRA.Mods.Common/Traits/SelectionDecorations.cs

```diff
@@ -9,39 +9,81 @@
 #endregion
 
 using System.Collections.Generic;
+using System.Drawing;
 using System.Linq;
 using OpenRA.Graphics;
+using OpenRA.Mods.Common.Graphics;
+using OpenRA.Traits;
 
-namespace OpenRA.Traits
+namespace OpenRA.Mods.Common.Traits
 {
-	public class SelectionDecorationsInfo : ITraitInfo
+	public class SelectionDecorationsInfo : ITraitInfo, ISelectionDecorationsInfo
 	{
 		public readonly string Palette = "chrome";
 
+		[Desc("Visual bounds for selection box. If null, it uses AutoSelectionSize.")]
```

Comment:
Also I think it should default to `Selectable.Bounds` (might even save some work in `Actor`'s constructor).